src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramOptionsTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telegram\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Bridge\Telegram\TelegramOptions;
+
+final class TelegramOptionsTest extends TestCase
+{
+    public function testAnswerCallbackQuery()
+    {
+        $sut = new TelegramOptions();
+
+        $returnedSut = $sut->answerCallbackQuery('123', true, 1);
+
+        $this->assertSame($sut, $returnedSut);
+        $this->assertEquals(
```

Comment:
```suggestion
        $this->assertSame(
```

Reply:
I am not sure it is a good idea for simple arrays, as if order of keys changes, arrays are still equals, though assertSame will fail. Is that ok?

Reply:
I am using `assertSame` whenever possible, even if that means I need to change/rearange my expectation.

Reply:
Quite often it is a good approach, I agree. Though in such a case, when comparing arrays and not actually caring about the order of keys in it, `assertEquals` is a good one to use.